crates/core/src/surfnet/svm.rs

```diff
@@ -641,6 +641,30 @@ impl SurfnetSvm {
         }
     }
 
+    pub fn reset_account(&mut self, pubkey: &Pubkey) -> SurfpoolResult<()> {
+        // Get the existing account if it exists
+        if let Some(account) = self.get_account(pubkey) {
+            // Remove from indexes
+            self.remove_from_indexes(pubkey, &account);
+        }
+
+        // Create empty account
+        let empty_account = Account {
+            lamports: 0,
+            data: vec![],
+            owner: solana_sdk_ids::system_program::id(),
+            executable: false,
+            rent_epoch: 0,
+        };
+
+        // Set the empty account
+        self.inner
+            .set_account(*pubkey, empty_account)
+            .map_err(|e| SurfpoolError::set_account(*pubkey, e))?;
```

Comment:
Rather than creating an empty account, you should be able to just do `Account::default()`:
```rust
 self.inner
            .set_account(*pubkey, Account::default())
            .map_err(|e| SurfpoolError::set_account(*pubkey, e))?;
```